### DISCUSSION TITLE
[Feature] Delete "motion photo" part of photo

### DISCUSSION THREAD

**Discussion by pe1uca** [immich-app/immich#13440] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I'd like a way to easily delete the motion part of a photo, since some photos don't really need it.  

For example I took this one and the motion doesn't really add anything to it, it just takes 4MB more of space compared to another one which didn't got the motion.  
<details>
  <summary>Example photo</summary>
  
  ![Example MP](https://github.com/user-attachments/assets/d15f13d4-fb3d-4ebc-915b-eed83a985ae1)
  
</details>

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4476
